biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -968,6 +983,7 @@ subroutine LeafLayerPhotosynthesis(f_sun_lsl,         &  ! in
 
         !is there leaf area? - (NV can be larger than 0 with only stem area if deciduous)
         if ( laisun_lsl + laisha_lsl > 0._r8 ) then 
+	  if(bbb > 1.0_r8)then !only if stomata open larger than cuticular conductance
```

Comment:
@xuchongang , I don't follow the connection between the logic and its comment.   Can you help me clear this up?

The code is seeing if the minimum leaf (cuticular) conductance (bbb) is greater than 1.  But the comment implies that its comparing the stomata opening against cuticular conductance.... 

Since 1 umol/m2/s is very very small, it seems that this comparison is really about seeing if btran>0 ? I must be missing something.

Reply:
@xuchongang and I checked in on this, and agreed that we can remove this logic: "I basically try to avoid the cap by 1 umol/m2/s and avoid the cases of unstable solutions of gs. I agree that 1 umol/m2/s is very very small and this part of change is not critical for the bug fix. I understand your confusion and I think we can leave the code as it is to avoid this confusion."